content/minfraud/_includes/request.json

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Where is this used?  Also, the file is called `request.json`, but this contains response data

Reply:
This file has been remove